ocaml/tests/test_certificates.ml

```diff
@@ -216,6 +223,23 @@ let invalid_chain_cert_tests =
     "Validation of an unsupported certificate chain", `Quick, test_cert)
   corrupt_chain_certificates
 
+let hashables =
+  [ "foobar",
+    "C3:AB:8F:F1:37:20:E8:AD:90:47:DD:39:46:6B:3C:89:74:E5:92:C2:FA:38:3D:4A:39:60:71:4C:AE:F0:C4:F2"
+  ]
+
+let pp_hash_test =
+  List.map (fun (hashable, expected) ->
+    let test_hash hashable repr () =
+      let digest = Cstruct.of_string hashable
+      |> Nocrypto.Hash.digest `SHA256
+      in
+      Alcotest.(check string) "fingerprints must match" expected (Certificates.pp_hash digest)
+    in
+    Printf.sprintf {|Validation of hash printing of "%s"|} hashable,
```

Comment:
you haven't used `repr`

this suggests you have been silencing some warnings locally, which you ought to fix

Reply:
warnings are always silenced when using `make test`, across the whole toolstack.

I'll fix this now.